app/models/package_manager/maven.rb

```diff
@@ -59,13 +59,19 @@ def self.recent_names
 
     def self.project(name)
       sections = name.split(':')
+      path = sections.join('/')
+      versions = versions({ path: path })
+      return {} unless versions.present?
+
       {
         name: name,
-        path: sections.join('/'),
+        path: path,
         groupId: sections[0],
         artifactId: sections[1],
-        versions: versions(h),
+        versions: versions,
       }
+    rescue
+      {}
```

Comment:
I think passing back an empty Hash will cause an error further downstream. I don't think the rescue will help.

Reply:
`Base` checks `project.present?` - this is the same pattern used in other package manager classes to signal "do nothing"

Reply:
Could we pass back the Hash with an empty versions if the versions call fails?